test/ReactWrapper-spec.jsx

```diff
@@ -2051,6 +2051,12 @@ describeWithDOM('mount', () => {
       expect(parents.at(1).hasClass('bar')).to.equal(true);
       expect(parents.at(2).hasClass('bax')).to.equal(true);
     });
+
+    it('should work with components in the tree', () => {
+      const Foo = () => (<div className="bar" />);
+      const wrapper = mount(<Foo />);
+      expect(wrapper.find('.bar').parent()).to.equal(wrapper.find('Foo'));
```

Comment:
Maybe, not sure what should be at the right side. I haven't had a chance to look what it returns because it fails now.